engine/dlib/src/dlib/log.cpp

```diff
@@ -72,7 +72,7 @@ struct dmLogServer
 };
```

Comment:
In short, I needed these changes in order to use the functionality from our C code (Remotery)